src/app/layout/footer/footer.component.html

```diff
@@ -0,0 +1,12 @@
+<div class="d-block d-sm-none footer fixed-bottom container">
```

Comment:
-Przyciski nie powinny być w footerze. Należą do navbara. hmmm Nazwij cały komponent navbar-mobile.
-Jak sie otwiera usera to trochę za małe są te kafelki. 
![image](https://user-images.githubusercontent.com/32534930/221894179-8af213bb-f64a-433c-acab-aee2a3822b30.png)
Najlepiej jakbyśmy na wersji mobilnej użyły takiego sidenava :
![image](https://user-images.githubusercontent.com/32534930/221893838-3a42bec9-735c-4ffa-beeb-6224832ac99a.png)
-Możesz wtedy tego sidenava do odzielnego komponentu wyodrębnić (bootrstap powinien mieć taki komponent)

-Nie potrzebnie dodałam  footer.
- Te przyciski fajnie wygladają. Tylko Według mnie powinny być mniejsze. Bo są za duże w porównaniu do wolnego tła na mobilce, 

Podsumowując, navbar dla desktop i mobile zrobiłabym w oddzielnych komponentach.
-navbar-desktop
-navbar-mobile

Logo można dodac do sharedModule. Bo będzie używane w 2 komponentach.
![image](https://user-images.githubusercontent.com/32534930/221894951-80773e1a-d329-47da-bbf5-561e96d4aea1.png).

Jak masz inny pomysł to daj znać :D